goverge/coverage.py

```diff
@@ -133,6 +138,9 @@ def generate_package_coverage(
     if tag:
         options.append("-tags={}".format(tag))
 
+    if custom:
+        options += custom
```

Comment:
Will this add a space to separate the given custom flags from whatever was previous?